doc/source/drivers/vector/xodr.rst

```diff
@@ -0,0 +1,198 @@
+.. _vector.xodr:
+
+XODR -- OpenDRIVE Road Description Format
+=========================================
+
+.. shortname:: XODR
+
+.. build_dependencies:: libOpenDRIVE >= 0.5.0, GEOS
+
+This driver provides read access to road network elements of OpenDRIVE datasets.
+
+`OpenDRIVE <https://www.asam.net/standards/detail/opendrive/>`_ is an open industry format for lane-detailed description of road networks, commonly used in applications of the automotive and transportation systems domains. It bundles polynomial, continuous road geometry modelling with all necessary topological links and semantic information from traffic-regulating infrastructure (signs and traffic lights).
+
+Driver capabilities
+-------------------
+
+.. supports_georeferencing::
+
+Specification version
+---------------------
+
+The currently supported OpenDRIVE version is 1.4 and basically depends on what is provided by libOpenDRIVE_. 
+
+.. _libOpenDRIVE: https://github.com/pageldev/libOpenDRIVE/
+
+Supported OpenDRIVE elements
+++++++++++++++++++++++++++++
+
+The XODR driver exposes OpenDRIVE's different road elements as separate layers by converting geometric elements into 3-dimensional OGR geometry types. The following _`layer types` are currently implemented:
+
+* *ReferenceLine*: Road reference line (``<planView>``) as :cpp:class:`OGRLineString`.
+* *LaneBorder*: Outer road lane border as :cpp:class:`OGRLineString`.
+* *Lane*: Polygonal surface (TIN) of the lane mesh as :cpp:class:`OGRTriangulatedSurface`.
+* *RoadMark*: Polygonal surface (TIN) of the road mark mesh as :cpp:class:`OGRTriangulatedSurface`.
+* *RoadObject*: Polygonal surface (TIN) of the road object mesh as :cpp:class:`OGRTriangulatedSurface`.
+* *RoadSignal*: Polygonal surface (TIN) of the road signal mesh as :cpp:class:`OGRTriangulatedSurface`.
+
+Spatial reference
+-----------------
+
+By definition, OpenDRIVE geometries are always referenced in a Cartesian coordinate system which defaults to having its origin at ``(0, 0, 0)``. If real-world coordinates are used, the OpenDRIVE header provides a PROJ.4 string with the definition of a projected spatial reference system:
+
+::
+
+  <header ...>
+    <geoReference><![CDATA[+proj=tmerc +lat_0=0 +lon_0=9 +k=0.9996 +x_0=500000 +y_0=0 +datum=WGS84 +units=m +no_defs]]></geoReference>
+  </header>
+
+The XODR driver uses this PROJ definition as spatial reference for creation of all OGR geometry layers. 
```

Comment:
Note: I assume you can't easily change the format. But it would be much better to use a WKT CRS string instead (cf https://www.ogc.org/standard/wkt-crs/)

Reply:
As you already assumed, this currently cannot be changed in OpenDRIVE. The `<geoReference>` tag has been introduced many years ago as part of the dataset's `<header>` with version 1.4 – this was already a novelty – and is still persisting in the recently released version 1.8. The OpenDRIVE standard defines it to hold a [PROJ projection string](https://proj.org/en/9.4/usage/projections.html). I will forward your hint regarding preferred usage of WKT CRS strings to the committee.